### PR TITLE
[INFRA/IIGO] Moving constants at top of judiciary to params

### PR DESCRIPTION
**Pull request by Neelesh99**
# Summary

Addressing Issue #262, moving constants defined in judiciary to params file.

## Test Plan

Made appropriate adjustments to tests

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

File: internal/server/iigointernal/judiciary_test.go [hunk 1]
```diff
@@ -427,7 +427,7 @@ func TestApplySanctions(t *testing.T) {
 				{
 					ClientID:     shared.Team1,
 					SanctionTier: roles.SanctionTier1,
-					TurnsLeft:    sanctionLength,
+					TurnsLeft:    2,
```
Comment (lhl2617) [inline internal/server/iigointernal/judiciary_test.go:432]: tests shouldn't break when the defaults in params are changed. you should be injecting these params into your functions.

Comment (Neelesh99) [reply internal/server/iigointernal/judiciary_test.go:432]: Fair enough, I think the functions are injected with these params, the test was poorly written. Refactored to use injected values.

**Review by NGovani — APPROVED** [review]
It looks like you've fixed LH's issues. If this passes tests feel free to merge it in

**Pull request merged**